### DISCUSSION TITLE
Raw price

### DISCUSSION THREAD

**Discussion by rolinbos** [duncanmcclean/simple-commerce#1161] [Help]
Is it possible to get the raw price of a product in Antlers? I want to do some math with the price, but it's now a string when I want to do some math with the price.

**Comment by duncanmcclean** [comment]
You should be able to do `{{ price | raw }}`.

**Reply by rolinbos** [reply]
It's displaying the same as ```{{ price }}``` I'm using the product variant. Is it possible that it only works for products that are not variants?

This is the code:
```{{ price | raw }} {{ price }}```
This is the output:
![Screenshot 2024-10-02 at 09 05 28](https://github.com/user-attachments/assets/6374e923-6787-41d3-a875-d16e59ad06e5)

I expect that the output would be:
€49,99 4999

**Reply by duncanmcclean** [reply]
Hmm, the `raw` modifier should give you `4999`. Can you do `{{ price | dump }}` - does it give you a string or a `Value` object?

**Reply by rolinbos** [reply]
It gives a string back.
![Screenshot 2024-10-02 at 15 56 26](https://github.com/user-attachments/assets/defa82d8-43d9-4fc9-8a6f-a83cd917795b)

**Reply by duncanmcclean** [reply]
Oh, it should probably be a `Value` object instead, since that'll let you get the raw value too.

I'm spending most of my free time working on the [next version of Simple Commerce](https://duncanmcclean.com/future-of-simple-commerce) at the moment, so I'll add it to my list. For now, you could probably achive it with a custom modifier which removes the symbols.
